src/ResourceManager/Profile/Commands.Profile.Test/AzureRMProfileTests.cs

```diff
@@ -733,6 +733,7 @@ public void SavingProfileWorks()
       ""AzureKeyVaultServiceEndpointResourceId"": null,
       ""GraphEndpointResourceId"": null,
       ""DataLakeEndpointResourceId"": null,
+      ""BatchEndpointResourceId"": null,
```

Comment:
can we add in tests that verify this value - we have tests that check the output type, and tests that check the value that is saved in the context

Reply:
Okay, I have updated the `TypeConversionTests` as well (I hadn't realized they were there earlier).

Can you point me to the test classes/methods for any other tests which I should update?